examples/Ethernet/CosaSNMP/CosaSNMP.ino

```diff
@@ -173,10 +178,11 @@ OutputPin sd(Board::D4, 1);
 #endif
 
 // Network configuration
-#define IP 192,168,1,100
-#define SUBNET 255,255,255,0
-#define GATEWAY 192,168,1,1
+// If WS5100 board has a MAC sticker then use here instead of this default
 static const uint8_t mac[6] __PROGMEM = { 0xde, 0xad, 0xbe, 0xef, 0xfe, 0xed };
+// If no DHCP set these, uncomment in var definitions, and switch to ethernet.begin() below
```

Comment:
Much cleaner - can you point me at somewhere you have done this (or something similar) and I'll update to maintain consistency with the way things are already done in Cosa.